controller/image.yaml

```diff
@@ -73,11 +73,11 @@ modules:
     - name: rhdm-7-image
       git:
         url: https://github.com/jboss-container-images/rhdm-7-image.git
-        ref: master
+        ref: main
```

Comment:
see my comment on the jira about the other repos.

Reply:
Ok

Reply:
Done